### DISCUSSION TITLE
[Application] FILPoll

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#113] [Project Showcase]
[FILPoll](https://filpoll.io) built by DMOB, is a polling tool for gauging community sentiment on FIPs. It tallies poll results based on a user's FIL token holdings, a miner's storage mining power, the amount of data stored by storage clients, or the addresses of verified core developers, app developers, and ecosystem collaborator organizations. 
It is currently in alpha testing phase and please leave a comment below if you have any feedback, suggestion,s or feature requests!

**Comment by neondragon** [comment]
The tool counts raw power, and not adjusted power. I would have expected adjusted power to be counted, as it corresponds to pledge collateral. What is the rationale for counting raw power only?

**Reply by mxz9102** [reply]
I think adjusted power makes more sense here since it indicates the amount of the collateral.
